apps/contracts/package.json

```diff
@@ -2,7 +2,6 @@
     "name": "contracts",
     "private": true,
     "scripts": {
-        "dev": "yarn compile && hardhat node & yarn deploy:zkgroups --network localhost",
```

Comment:
Does it make sense to keep the command to apps the apps without Docker?

Reply:
Yes, I have added it as another script "local-node". This was to avoid starting it up with `yarn dev` and have port collission with docker ganache

Reply:
Great, I didn't see that script

Reply:
Ah no! I added it later, sorry.